web/client/examples/viewer/plugins/mobile.jsx

```diff
@@ -121,6 +124,10 @@ module.exports = {
                                 crs={(props.mousePositionCrs) ? props.mousePositionCrs : props.map.projection} />
                         </Settings>
                     </Section>
+                    <Section eventKey="fullscreen" header={<Message msgId="settings-scr" />}>
```

Comment:
We don't need a section for the `fullscreen` options. Put the `<FullScreen>` component in the `<Settings>` section.